### PR TITLE
Lenel IAM

### PR DESCRIPTION
**Pull request by dansterenson**
Lenel IAM integration

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38910

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by dansterenson** [comment]
@dantavori Ready for review.

File: Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+  type: 8
+- display: Lenel API Version (e.g. 1.0). Only specify this value to use an endpoint
+    version other than the latest.
+  name: api_version
+  defaultvalue: "1.0"
+  type: 0
+  required: true
+- display: Application-id
+  name: application_id
+  defaultvalue: f71524e0-2b15-42b8-bb3d-34403695d417
```
Comment (dantavori) [inline Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml:55]:
```suggestion
```

File: Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+  required: true
+  type: 0
+- defaultvalue: User Profile - IAM Lenel (Outgoing)
+  display: Outgoing Mapper
+  name: mapper_out
+  required: true
+  type: 0
+description: An Identity and Access Management integration template.
+display: LenelIAM
+name: LenelIAM
```
Comment (dantavori) [inline Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml:70]:
```suggestion
display: Lenel IAM
name: Lenel IAM
```

File: Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+  display: Incoming Mapper
+  name: mapper_in
+  required: true
+  type: 0
+- defaultvalue: User Profile - IAM Lenel (Outgoing)
+  display: Outgoing Mapper
+  name: mapper_out
+  required: true
+  type: 0
+description: An Identity and Access Management integration template.
```
Comment (dantavori) [inline Packs/Lenel/Integrations/LenelIAM/LenelIAM.yml:68]:
```suggestion
description: Integrate with Lenel services to perform Identity Lifecycle Management
  operations.
```

File: Packs/Lenel/Integrations/LenelIAM/README.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
+An Identity and Access Management integration template.
```
Comment (dantavori) [inline Packs/Lenel/Integrations/LenelIAM/README.md:1]:
```suggestion
Integrate with Lenel services to perform Identity Lifecycle Management operations.
```

File: Packs/Lenel/Integrations/LenelIAM/README.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+| IAM.Vendor.id | String | The employee's user ID in the app. | 
+| IAM.Vendor.instanceName | string | Name of the integration instance. | 
+| IAM.Vendor.success | Boolean | When true, indicates that the command was executed successfully. | 
+| IAM.Vendor.username | String | The employee's username in the app. | 
+
+
+#### Command Example
+``` !iam-create-user user-profile=`{"email": "john.doe@example.com", "givenname": "test", "surname": "test"}`  ```
+
+#### Human Readable Output
```
Comment (dantavori) [inline Packs/Lenel/Integrations/LenelIAM/README.md:66]: please add outputs (also in other commands)

File: Packs/Lenel/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Lenel",
+    "description": "LS OpenAccess service that can be used to manage",
```
Comment (dantavori) [inline Packs/Lenel/pack_metadata.json:3]:
```suggestion
    "description": "Integrate with Lenel services to perform Identity Lifecycle Management operations.",
```

**Review by dantavori — CHANGES_REQUESTED** [review]
sorry for the late response (we recently reduced the high effort on IAM)
approving docs since it's based on the template.

**Pull request closed without merging**